### PR TITLE
SKALE-2894 fixed Rinkeby compatibility issues

### PR DESCRIPTION
**Pull request by sergiy-skalelabs**
Fixed Rinkeby compatibility issues

### CODE REVIEW

File: npms/skale-ima/index.js [hunk 1]
```diff
@@ -137,6 +137,52 @@ async function get_contract_call_events( joContract, strEventName, nBlockNumber,
...
+            delete rawTx.chainId;
+            joOpts = joOpts || { };
+            joOpts.chain = "rinkeby";
+            break;
+        case 5:
+            delete rawTx.chainId;
+            joOpts = joOpts || { };
+            joOpts.chain = "goerli";
+            break;
+        case 2018:
```
Comment (payvint) [inline npms/skale-ima/index.js:166]: What is the secret number 2018?:)

**Review by payvint — APPROVED** [review]
Cool!

File: npms/skale-ima/index.js (hunk 1 above)
Comment (sergiy-skalelabs) [reply npms/skale-ima/index.js:166]: **2018** is **chainId** of network called **dev**. It's listed in common set of networks with **Rinkeby** and **Ropsten**.

**Review by cstrangedk — APPROVED** [review]

**Review by christineRPM — APPROVED** [review]
Looks good! Excited for this!

**Review by kladkogex — APPROVED** [review]

**Pull request merged**